pkg/ibmvpc/examples/out/explain_out/tgwAnotherEnableDefaultDifFile_all_vpcs_explain.txt

```diff
@@ -1,5 +1,5 @@
-Explaining connectivity from vsi11-ky to vsi21a-ky
-==================================================
+Explaining connectivity from test-vpc1-ky/vsi11-ky to test-vpc2-ky/vsi21a-ky
```

Comment:
Also here. Perhaps have 
Explaining connectivity from vsi11-ky (test-vpc1-ky/vsi11-ky) to vsi21a-ky (test-vpc2-ky/vsi21a-ky)